source/_integrations/stream.markdown

```diff
@@ -13,52 +13,20 @@ ha_codeowners:
 ha_domain: stream
 ---
 
-The `stream` integration provides a way to proxy live streams through Home Assistant. The integration currently only supports proxying H.264 source streams to the HLS format and requires at least FFmpeg >= 4.
+The `stream` integration provides a way to proxy live streams through Home Assistant. Most users should not need configure anything or interface with the component directly since it is an internal component used by the [`camera`](/integrations/camera) integration.
```

Comment:
```suggestion
The `stream` integration provides a way to proxy live streams through Home Assistant. Most users should not need to configure anything or interface with the component directly since it is an internal component used by the [`camera`](/integrations/camera) integration.
```